### ISSUE TITLE
Trainer Pokemobs fight me, not my pokemob.

### ISSUE THREAD

**Issue by pokiiiiiii** [Pokecube-Development/Pokecube-Issues-and-Wiki#1095]
What happens: everytime i start a fight with a trainer his pokemobs only focus me, if i start and go instant switch in the creative mode then he fight my pokemob. i tried different things, uninstalled, deactivate all mods, but nothing change. i have the newest version from pokecube aio, forge and play in 1.19.2.  and i dont really use other mods.

Edit: ive tried the other versions of pokecube, int he last versions 1,2,3 last weeks, i cant fight in general with other trainers there, but in 3.22.4 they start fight but the trainer focus instant me with his face, and if i walk arround him, his face follow my steps so, if the start fight the player is the target.  and sometimes, my pokemon run instant away to fight a random wild pokemon if the start fight.

#### Steps to reproduce:

1. i rightklick a trainer with a filled pokecube, charmander lvl5.
2. we send out the pokemobs.
3. all of his pokemob one after one  attack me, and my pokemob freekills all his pokemons
...

____
#### Affected Versions (Do *not* use "latest"): Replace with a list of all mods you have in.

- Pokecube AIO:
- Minecraft: 1.19.2
- Forge: 43.1.58

**Comment by Nathan22211** [comment]
yeah I think the premise of this mod over pixelmon is that the pokemon fit more so into the survival aspect of the game, so I think the trainers are meant to be like hostile mobs

**Comment by Imploder** [comment]
Pokecube AIO: 3.22.4
Minecraft: 1.18.2
Forge: 40.2.1

This is happening to me too, in a battle the enemy pokemobs will attack me instead of my pokemob. I would like to learn if this is intentional or not since it does make battles easy. Unlike OP however I have added pokecube to my large kitchen sink pack so many other mods are present.

Edit:
Just tested on the same versions with Pokecube as the only mod installed and the outcome is the same.

**Comment by sergy452** [comment]
Did you managed to fix the problem at the end?
